modules/serverless-service-mesh-enable-sidecar-injection.adoc

```diff
@@ -5,13 +5,18 @@
 [id="serverless-enable-sidecar_{context}"]
 = Enabling sidecar injection for a Knative service
 
-You can add an annotation to the Service resource YAML file to enable sidecar injection for a Knative service.
+You can add the `sidecar.istio.io/inject="true"` annotation to a Knative service to enable sidecar injection that service.
+
+[IMPORTANT]
+====
+Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
```

Comment:
BTW. this note is also present in https://github.com/openshift/openshift-docs/blob/de8fe50df99d892f09c760017173dc3f51b350f8/modules/serverless-ossm-setup.adoc , so it appears twice on https://deploy-preview-32929--osdocs.netlify.app/openshift-enterprise/latest/serverless/networking/serverless-ossm.html  quite close together  (one in "Using Service Mesh with OpenShift Serverless" , the other in "Enabling sidecar injection for a Knative service"

Reply:
Good catch, I'll remove it from _Using Service Mesh with OpenShift Serverless_ since it's specific to sidecar injection

Reply:
I think I actually updated this in another PR for service mesh that Chris is reviewing atm: https://github.com/openshift/openshift-docs/pull/33179
There's a lot of stuff moving around in the service mesh docs because of the new way of integrating it, so I'll leave it for that PR to prevent a rebasing nightmare I think

Reply:
Note to self: it's actually in the `serverless-ossm-setup` module, so remove it there also

Thanks for the input

Reply:
Removed from here https://github.com/openshift/openshift-docs/pull/32929/files#diff-37623fdc0689479c3749a99e4f2826dee743dd7e058706182c6fab4c3965febbL24-L28